browser/templatetags/app_filters.py

```diff
@@ -141,3 +141,11 @@ def get_affiliates(institution):
     #  absolute URL.
 
     return qs
+
+
+@register.filter
+def get_positions(person, role_type):
+    if role_type == 'All':
+        return Position.objects.filter(person=person).distinct('year', 'id').order_by('year', 'id')
```

Comment:
are there objects that are returned twice or why do you do an distinct on year and id?

Reply:
The <b>distinct</b> part was not required so I have removed it from the query. Also I have removed <b>id</b> from the <b>order-by</b> part of the query.